server/template.go

```diff
@@ -165,6 +165,10 @@ func init() {
 #pull-request-labeled ` + "`{{.GetLabel.GetName}}`" + ` by {{template "user" .GetSender}}
 `))
 
+	template.Must(masterTemplate.New("pullRequestMentionNotification").Funcs(funcMap).Parse(`
+{{template "user" .GetSender}} mentioned you on [{{.GetRepo.GetFullName}}#{{.GetPullRequest.GetNumber}}]({{.GetPullRequest.GetHTMLURL}}) - {{.GetPullRequest.GetTitle}}:
+>{{.GetPullRequest.GetBody | trimBody | replaceAllGitHubUsernames}}`))
```

Comment:
Not 100% sure, but if the body had several line breaks, this may not show the body correctly quoted. For example, if the body was:
```
A

B
```
This will be rendered as:
```
> A

B
```
which will only quote the first line.

Reply:
@larkox That is a correct observation. But the issue also exists with all other notifications. Would you mind filling an issue and this get's fixed in a speperate PR?